hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/HollowAccountFinalizationSuite.java

```diff
@@ -151,7 +153,8 @@ final Stream<DynamicTest> hollowAccountCompletionWithEthereumTransaction() {
                             .via(TRANSFER_TXN_2);
 
                     final var op3 = getAliasedAccountInfo(evmAddress)
-                            .has(accountWith().key(SECP_256K1_SOURCE_KEY).noAlias());
+                            .has(accountWith().key(SECP_256K1_SOURCE_KEY).noAlias())
+                            .logged();
```

Comment:
(what does this additional `.logged()` check?  was it just omitted before or is there something new to check?)

Reply:
removed.  just wanted to make sure that the account had the right key.